shared/components/home/card.tsx

```diff
@@ -23,11 +24,11 @@ const HomeCard = (props: {
       </h1>
       <p className="text-white font-light px-3 mt-3 text-xs">{props.desc}</p>
       <div className="text-center mt-10 mb-5">
-        <a href={`${props.link}`} target="_blank" rel="noopener noreferrer">
+        <Link href={`/events`}>
           <button className="bg-black border rounded-full text-white text-xs px-5 py-1 border-white focus:outline-none">
             Know More
           </button>
-        </a>
+        </Link>
```

Comment:
also @AdityaGupta0001 why did you use Link instead of a tag?

Reply:
Since we don't need to refresh the page everytime the user clicks "Know more" we use Link tag to just redirect to another page of the website.

Reply:
Great answer @AdityaGupta0001

Reply:
Do the rest of the fixes please @AdityaGupta0001

Reply:
Done the changes @utkarshini14

Reply:
<img width="531" alt="image" src="https://github.com/srm-kzilla/srmkzilla-net/assets/74819565/4feb4c9c-0abf-4803-829d-77cee213f972">

@AdityaGupta0001 you have to click on request review again everytime you make the changes, the little circle button

Reply:
Done